modules/e2ee/Context.js

```diff
@@ -270,21 +316,41 @@ export class Context {
         if (this._cryptoKeyRing[keyIndex]) {
             const counterLength = 1 + ((data[encodedFrame.data.byteLength - 1] >> 4) & 0x7);
             const signatureLength = data[encodedFrame.data.byteLength - 1] & 0x80
-                ? this._signatureOptions.byteLength : 0;
+                ? this._signatureOptions.byteLength + 1 : 0;
```

Comment:
this is the length of the signature and the one byte for the count. Maybe use a named constant for the field length?

Reply:
reviewer note: the definition of signatureLength is different in encode and decode function